src/renderer/components/utils/menu/AccessibleMenu.tsx

```diff
@@ -134,11 +134,13 @@ export default class AccessibleMenu extends React.Component<Props, State> {
         }
     }
 
-    private handleFocus(event: any) {
+    private handleFocus(event: Event) {
         const focusedNode = event.target;
 
         if (this.containerRef
             && this.containerRef.current
+        // WHY ?
+        // @ts-ignore
```

Comment:
Simply replace the line `const focusedNode = event.target;` with `const focusedNode = event.target as HTMLElement;` and remove the `@ts-ignore` comment.

Reply:
same above

Reply:
`Event.target` is a generic DOM property, which does not only apply to HTML (also SVG, XML, etc.). However, in Thorium we know that the UI is implemented in HTML markup (with the odd SVG image here and there), so we can safely cast the appropriate type for better static / compile-time checking.
https://developer.mozilla.org/en-US/docs/Web/API/Event/target
https://developer.mozilla.org/en-US/docs/Web/API/Event

Reply:
d7e13e2

👍